integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/Hooks.java

```diff
@@ -59,10 +59,13 @@ public static void initializeLocalTestConfig() {
                 Properties prop = System.getProperties();
                 prop.load(inputStream);
                 System.setProperties(prop);
+                WEBHOOK_ID = Utils.getSystemProperty("webhook.site.uuid");
```

Comment:
This code branch is executed only when `localconfig.properties` is defined. However for CI we don't use that file, so for CI this code is skipped, causing crash for `deleteWebhookSiteRequestHistory()`

Reply:
Oh! okk. So, that means some other approach has to be followed, so that it is compatible with both the CI and local setup.

Reply:
Yes